lib/cli-engine.js

```diff
@@ -553,6 +554,9 @@ CLIEngine.prototype = {
                     if (warnIgnored) {
                         results.push(createIgnoreResult(filename));
                     }
+                    if (options.verbose) {
+                        log.info("Ignoring: " + filename);
```

Comment:
That's a good thought. Currently it is output to `stdout`. Perhaps we should use `console.warn` instead for this case?